.travis.yml

```diff
@@ -0,0 +1,53 @@
+language: node_js
+node_js:
+  - "lts/*"
+services:
+  - docker
+
+before_install:
+  - if [[ -v DOCKERHUB_TOKEN ]]; then echo "${DOCKERHUB_TOKEN}" | docker login -u "icdevops" --password-stdin; fi
+
+script:
+  # Audit npm packages. Fail build whan a PR audit fails, otherwise report the vulnerability and proceed.
+  - if [ "${TRAVIS_PULL_REQUEST}" != "false" ]; then npx audit-ci --low; else npx audit-ci --low || true; fi
+  - npm run lint
+  - docker build --rm -t "quay.io/razee/impersonation-webhook:${TRAVIS_COMMIT}" .
```

Comment:
do you have a strong preference on `impersonation-webhook`? When i created the quay.io image registry i used `impersonationwebhook`. I can create a new one if you really want, but i was thinking youd use `impersonationwebhook` to match the repo name

```suggestion
  - docker build --rm -t "quay.io/razee/impersonationwebhook:${TRAVIS_COMMIT}" .
```

Reply:
I just thought that `impersonationwebhook` is a bit hard to read. Long word :) Yeah if you can change the repo name, that would be great! If not, I can update this.

Reply:
ok updated: https://quay.io/repository/razee/impersonation-webhook?tab=info